challenges/2021-04-27-sum-arrays/challenge.md

```diff
@@ -0,0 +1,45 @@
+# Sum Arrays
+
+Given an array of values, return the sum of the values.
+
+## Business Rules/Errata
+
+- Input must be an array.
+- The array may be nested more than one level.
+- All values must be integers.
+- Solutions shall not use built in methods to flatten the array to one-dimension.
+- You will need 'rspec' if you want to run the tests. From the project `src/` folder run either:
```

Comment:
I might move the system requirements/install instructions to the end, after Examples. Maybe include a link to instructions for installing gem/bundler? (Ruby folks probably have this already).